wasm-track-stdlib.md

```diff
@@ -0,0 +1,278 @@
+WebAssembly code can only access external state or functions explicitly provided as imports. Similarly, code in the external host environment can only access state and functions defined within WebAssembly that are explicitly exported.
```

Comment:
If you rename this file to README.md or similar, it will show up automatically in a lot of different places.